### ISSUE TITLE
Bottlerocket/RPM package

### ISSUE THREAD

**Issue by chadbrewbaker** [uutils/coreutils#1593]
Amazon has a new Rust heavy distro https://github.com/bottlerocket-os/bottlerocket

Need a RPM release package for uutils/coreutils, and to test it on Bottlerocket. See https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/coreutils

**Comment by sylvestre** [comment]
Sweet.
Coreutils rust isn't ready for production but this would be a nice to have :+1:

**Comment by beedell-roke** [comment]
Please reopen. `.RPM` is as necessary as `.deb` nowadays, for Red Hat and *SUSE's OSes.

**Comment by sylvestre** [comment]
patches welcome

**Comment by oech3** [comment]
I don't agree with supporting distro specific packages. It would make our deployment 100+.
We should have scripts for re-packaging if it is absolutely needed.

**Comment by beedell-roke** [comment]
@oech3, I believe that many projects utilise the Open Build Service to do this for myriad distributions, if that is relevant.

**Comment by oech3** [comment]
But I don't think this repo should have such complexity on the CI.

**Comment by oech3** [comment]
Do we still want to have this on this repository? I really don't want to do that.
